community-plugins.json

```diff
@@ -5563,5 +5563,12 @@
         "description": "This is a markdown export plugin for Obsidian.",
         "author": "bingryan",
         "repo": "bingryan/obsidian-markdown-export-plugin"
+    },
+        {
+        "id": "obsidian-gpt",
+        "name": "Obsidian GPT",
```

Comment:
Hi, we're asking plugins to avoid using the word "Obsidian" in their plugin name to avoid over-saturating the brand name and reserve the naming for first-party plugins (like Obsidian Publish, Obsidian Sync, etc).